src/services/types.ts

```diff
@@ -130,3 +136,11 @@ export type ReferrerReward = {
   timestamp: number;
   amount: string;
 };
+
+export type ReferrerSetReferrer = {
```

Comment:
Shouldn't it be named `ReferrerSetReferral`? I guess Referrer couldn't set another Referrer